packages/sst/src/constructs/NextjsSite.ts

```diff
@@ -111,14 +136,21 @@ export class NextjsSite extends SsrSite {
   constructor(scope: Construct, id: string, props?: NextjsSiteProps) {
     super(scope, id, {
       buildCommand: [
-        "npx --yes open-next@2.2.0 build",
+        "npx --yes open-next@2.2.1 build",
         ...(props?.regional?.experimentalStreaming ? ["--streaming"] : []),
+        ...(props?.dangerous?.disableDynamoDbCache ? ["--dangerously-disable-dynamodb-cache"] : []),
+        ...(props?.dangerous?.disableIncrementalCache ? ["--dangerously-disable-incremental-cache"] : []),
       ].join(" "),
       ...props,
     });
 
-    this.createRevalidationQueue();
-    this.createRevalidationTable();
+    
+    if (!props?.dangerous?.disableIncrementalCache) {
+      this.createRevalidationQueue();
+    }
+    if (!props?.dangerous?.disableDynamoDbCache || !props?.dangerous?.disableIncrementalCache) {
```

Comment:
If I only want to disable DDB cache, but have Incremental Cache working, do I need to create the DDB table?

Reply:
No this is not necessary, it only means that `next/cache` won't work

Reply:
I may be confused on the second or statement. Because if I use `disableDynamoDbCache: true` and leave `disableIncrementalCache` as false, this will create the DDB table?

Reply:
You're right, i need to be more careful with copilot sometimes. It's missing parenthesis